src/screens/organization/organization-profile.js

```diff
@@ -0,0 +1,198 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { StyleSheet, RefreshControl } from 'react-native';
+import { ListItem } from 'react-native-elements';
+import ActionSheet from 'react-native-actionsheet';
+import {
+  ViewContainer,
+  OrgProfile,
+  SectionList,
+  ParallaxScroll,
+  LoadingMembersList,
+  EntityInfo,
+  UsersAvatarList,
+} from 'components';
+import { emojifyText, translate, openURLInView } from 'utils';
+import { colors, fonts } from 'config';
+import { getById, getMembers } from 'api/rest/providers/github/endpoints/orgs';
```

Comment:
Just for further consideration: it would be cool to just write :thinking: 
```js
import { getById, getMembers } from 'api';
```

More precisely, then it would be so:
```js
import { getOrganizationById, getOrganizationMembers } from 'api';
```

Proposal: instead of `getOrganizationById` maybe was simply `getOrganization`